Automattic/wp-calypso
# Jetpack Plans: Update the features listing

We have recently made some packaging changes to Jetpack plans and need to move some features from Professional to Premium (SEO Tools and Jetpack Concierge).

Test plan:
* Go to the /plans page for any Jetpack site.
* Confirm that the features listing changes are correctly reflected (i.e. "SEO Tools" and "Jetpack Concierge" are found under the Premium plan).
* Switch to "Monthly" billing.
* Confirm the same.
